ci_tools/process_results.py

```diff
@@ -25,6 +25,7 @@
 
     pyccel_folder = os.path.abspath('compare')
 
+
```

Comment:
I suppose this empty line was not intentional

Reply:
No it wasn't. Fixed : b008e889